### PR TITLE
LEAF-1533: Integrate New Nav Header

### PR DESCRIPTION
**Pull request by cameron-chilton**
Integration of newly designed global navigation into LEAF Request Portal Admin:
- New design navigation integrated to main.tpl, link hrefs pointing to correct targets
- Login/user menu container and icon added, functions in progress
- New css and js for global nav - navigation is now keyboard accessible (508)
- Saved original menu file in case needed for future reference
- Menu is responsive at mobile size

### CODE REVIEW

**Comment by cameron-chilton** [comment]
> Is that hamburger icon meant to be there? also when you click on any buttons in Nav the focused borders gets cut off
> Screen Shot 2020-05-26 at 7 45 13 PM
> Screen Shot 2020-05-26 at 7 45 21 PM

The hamburger icon was meant to be the start of a user menu, removed it for now.
Screen Shot 2020-05-27 at 9 13 28 AM

**Comment by cameron-chilton** [comment]
> > Is that hamburger icon meant to be there? also when you click on any buttons in Nav the focused borders gets cut off
> > Screen Shot 2020-05-26 at 7 45 13 PM
> > Screen Shot 2020-05-26 at 7 45 21 PM
> 
> The hamburger icon was meant to be the start of a user menu, removed it for now.
> Screen Shot 2020-05-27 at 9 13 28 AM

Added a little margin between the level one nav elements so that focus border is not cut off:
Screen Shot 2020-05-27 at 9 18 22 AM

File: LEAF_Request_Portal/admin/templates/menuORIGINAL.tpl [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<a href="../" class="usa-button site-button-outline-secondary">Main Page</a>
+<span id="headerMenu_container" class="leaf-position-relative">
```
Comment (dastan-rahimi) [inline LEAF_Request_Portal/admin/templates/menuORIGINAL.tpl:2]: Do we need this file?

File: LEAF_Request_Portal/admin/templates/menu.tpl [hunk 2]
```diff
@@ -1,49 +1,217 @@
...
+})
+
+// Add plus mark to li that have a sub menu
+$('li.lev2:has("ul") > a').append('<i class="fas fa-angle-down leaf-nav-icon"></i>');
+$('li.lev3:has("ul") > a').append('<i class="fas fa-angle-left leaf-nav-icon"></i>');
+
+
+
+
+// sub menu
```
Comment (thomasnguyen95) [inline LEAF_Request_Portal/admin/templates/menu.tpl:127]: Should clean up some of these comments.

**Pull request merged**